File: codewars/dart/7kyu/two_to_one.dart
void main(List<String> args) {
  print(longest('aretheyhere', 'yestheyarehere'));
  print(longest('loopingisfunbutdangerous', 'lessdangerousthancoding'));
  print(longest('inmanylanguages', 'theresapairoffunctions'));
}

String longest(String a, String b) {
  var lettersList = [for (var rune in (a + b).runes) String.fromCharCode(rune)];
  lettersList.sort();
  return lettersList.toSet().join();
}

File: codewars/python/7kyu/two_to_one.py
def longest(a1: str, a2: str) -> str:
    return ''.join(sorted(set(i for i in a1 + a2)))


if __name__ == '__main__':
    print(longest("aretheyhere", "yestheyarehere"))
    print(longest("loopingisfunbutdangerous", "lessdangerousthancoding"))
    print(longest("inmanylanguages", "theresapairoffunctions"))
